### PR TITLE
Add href to link on event creation page

### PR DESCRIPTION
**Pull request by cnasc**
# Description

The link on the event creation page wasn't clickable because it didn't have an href property. This PR adds that and a regression test.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by cnasc** [comment]
Oh. Hm, this doesn't quite work. I think maybe `EventUrl` should be refactored as a function rather than a component, then we can use it to construct the href.

File: tickets/src/components/content/CreateContent.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ export class CreateContent extends Component {
                     <SaveButton type="submit">Save Event</SaveButton>
                     <Text>
                       Your event link: <br />
-                      <Cta>
+                      <Cta href={lockAddress}>
```
Comment (benwerd) [inline tickets/src/components/content/CreateContent.js]: Should this open in a new tab?

Comment (cnasc) [reply tickets/src/components/content/CreateContent.js]: It probably should — I think an overall better approach will be to make `EventUrl` return an `<a>` tag with that baked in.

**Comment by julien51** [comment]
I think this is a better approach? https://github.com/unlock-protocol/unlock/pull/2850

**Comment by julien51** [comment]
I think this is a better approach? https://github.com/unlock-protocol/unlock/pull/2850

**Comment by cnasc** [comment]
Going to close this since we have other approaches in the works.

**Pull request closed without merging**